src/lib.rs

```diff
@@ -1979,11 +1986,15 @@ impl ByteOrder for BigEndian {
     #[inline]
     fn read_uint(buf: &[u8], nbytes: usize) -> u64 {
         assert!(1 <= nbytes && nbytes <= 8 && nbytes <= buf.len());
-        let mut out = [0u8; 8];
-        let ptr_out = out.as_mut_ptr();
+        let mut out = Align8([0u8; 8]);
```

Comment:
Is there perhaps a way to force alignment here without bumping the MSRV? Perhaps something like:

```rust
let mut out = 0u64;
let ptr_out = &mut out as *mut u64 as *mut u8;
```

Reply:
Great idea! Interestingly, using `u128` fails the build on Rust 1.12, even though the `byteorder_i128` feature is disabled.